_posts/2025-11-28-release-0_18_0.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: "Kroxylicious release 0.18.0"
+date: 2025-11-28 00:00:00 +0000
+author: "Keith Wall"
+author_url: "https://github.com/k-wall"
+# noinspection YAMLSchemaValidation
+categories: blog kroxylicious-proxy kroxylicious-operator releases
+tags: [ "releases", "kroxylicious-proxy", "kroxylicious-operator" ]
+---
+
+The Kroxylicious project is very pleased to announce the release
+of [Kroxylicious 0.18.0](https://github.com/kroxylicious/kroxylicious/releases/tag/v0.18.0). See
+the [Changelog](https://github.com/kroxylicious/kroxylicious/blob/main/CHANGELOG.md#0180) for a list of changes and
+summary of Deprecations, Changes and Removals.
+
+The main highlights of this release are:
+
+### Filter API Improvements
+
+* Filters now have a convenient mechanism to convert from topic ids [KIP-516](https://cwiki.apache.org/confluence/display/KAFKA/KIP-516%3A+Topic+Identifiers) to topic names.
+* For authenticated use-cases, filters now have access to a Subject API that provides convenient access to the authenticated principals established by the channel.  This is a building block
+  for filters that need to enforce authorization checks or want to write an audit trail of Kafka actions.
+
+### Authorizer Filter
+
+We've added a new Authorizer Filter that can authorize Topic operations using the same semantics as Kafka own Authorization API. This allows you to enforce authorizations rules at the proxy.
+
+### Azure KMS integration for Record Encryption
```

Comment:
I would bump this before the authorization section, since we think it's good to go.